README.md

```diff
@@ -27,11 +27,49 @@ TODO: Write documentation for how to build initrd locally
 
 See [Hacking](HACKING.md)
 
-# Integrating with Kernel Snap
+# Releasing
 
-When your initrd DEB package is built, you need to perform following steps to make it a part of kernel snap:
+The UC initrd is included in kernel snaps. However, the process to get
+it inside the kernel snap is not immediate and consists of a few
+steps. First, we need to build the `ubuntu-core-initramfs`
+debian package in the [snappy-dev/image
+PPA](https://launchpad.net/~snappy-dev/+archive/ubuntu/image) by
+following these steps:
 
-TODO: Write documentation for how to integrate initrd.img to your (custom or canonical) kernel 
+1. Update the changelog with latest changes since last release (use `dch -i` for this)
+1. Propose a PR to the repo with the new changelog, get it reviewed and merged
+1. Tag the repository with the new version
+1. Build the source package by running
+
+        git clean -fdx
+        dpkg-buildpackage -S -sa -d
+
+1. Compare with the latest package that was uploaded to the snappy-dev
+PPA to make sure that the changes are correct.  For this, you can
+download the .dsc file and the tarball from the PPA, then run debdiff
+to find out the differences:
+
+        wget https://launchpad.net/~snappy-dev/+archive/ubuntu/image/+sourcefiles/ubuntu-core-initramfs/<old_version>/ubuntu-core-initramfs_<old_version>.dsc \
```

Comment:
you can use  `dget` here on the dsc file, that will automatically fetch the tar.xz

Reply:
Ah, nice one, changed now